content/GettingStarted/SCALEReleaseNotes.md

```diff
@@ -38,30 +44,77 @@ weight: 10
+Attempting to update SCALE with `apt` or methods other than the SCALE web interface can result in a nonfunctional system.
+
+All auxiliary parameters can change between TrueNAS major versions due to security and development changes.
+We recommend removing all auxiliary parameters from TrueNAS configurations before upgrading.
+
+Systems with large numbers of attached disks are recommended to use the new Pool Creation wizard when creating or modifying a pool.
 
-To download an <file>.iso</file> file for installing or upgrading to SCALE 23.10 (Cobia), go to https://www.truenas.com/truenas-scale/ and click **Download**.
-SCALE major version upgrade process: [TrueNAS Upgrades]({{< relref "23.10Upgrades.md" >}})
+### Upgrade Paths (Anticipated)
 
-To update minor versions within SCALE 23.10 (Cobia), log in to your SCALE web interface and go to **System Settings > Update**.
-Manual update files are provided from https://www.truenas.com/truenas-scale/.
-SCALE minor version update process: [SCALE Community]({{< relref "UpdateSCALE.md" >}}) | [SCALE Enterprise]({{< relref "UpdateHASCALE.md" >}})
+See the <a href="https://www.truenas.com/software-status/" target="_blank">TrueNAS Software Status</a> page for recommendations about which software version to use based on your user type.
 
-### Cobia Unstable Nightly Images
+Update the system to the latest maintenance release of the installed major version before attempting to upgrade to a new TrueNAS SCALE major version.
 
-{{< hint type=warning >}}
-Nightly builds are considered experimental and highly unstable.
-Do not use a nightly build for anything other than testing and development.
+{{< hint type="warning" title="System Configuration File Compatibility" >}}
+System configuration files generated from releases before **22.12.4 (Bluefin)** are not compatible with 23.10 (Cobia).
+When available, update the system to **22.12.4 (Bluefin)**, resolve any migrations from deprecated services to replacement apps, and download a fresh system configuration file before attempting to upgrade.
 {{< /hint >}}
-Nightly images for TrueNAS SCALE are built every 24 hours, at around 2AM Eastern (EDT/EST) time.
-These images are made publicly available when they pass automated basic usability testing.
-This means that during times of heavy development, nightly images might be less frequently available.
-Online updates are created every 2 hours and are available in the SCALE UI online updating page.
-* [ISO Installation Files](https://download.truenas.com/truenas-scale-cobia-nightly/ "SCALE 23.10 (Cobia) Nightly .iso files")
 
-## 23.10 Component Versions
+{{< columns >}}
+**TrueNAS SCALE**
+
+```mermaid
+flowchart LR
+
+A[Angelfish 22.02.4] --> C
+B[CORE 13.0-U5.3] --> C
+C[Bluefin 22.12.4] --> D
+D[Cobia 23.10.0]
+```
+
+<--->
+**TrueNAS SCALE Enterprise**
+
+```mermaid
+flowchart LR
+A(Current 22.12 Bluefin version) --> B[Bluefin 22.12.4] --> C[Cobia 23.10.0]
+```
+
+{{< enterprise >}}
```

Comment:
This looks weird in the rendered version with it nested under the SCALE Enterprise migration path.
Suggest taking this out of the second column.